src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
My ongoing work on this is here: https://github.com/cowlicks/privacybadgerchrome/tree/firefox-compat

I'm going to dive in to implementing this today.